clientgui/ViewWork.cpp

```diff
@@ -745,6 +766,22 @@ wxString CViewWork::OnListGetItemText(long item, long column) const {
             case COLUMN_STATUS:
                 strBuffer = work->m_strStatus;
                 break;
+            case COLUMN_ESTIMATEDCOMPLETION:
+                //if ((work->m_fCPUTime > 0) && (work->m_strStatus.IsSameAs("Running"))) {
+                if ((work->m_fCPUTime > 0) && (work->m_strStatus.Contains("Running"))) {
```

Comment:
This check will work for English locale only.
See https://github.com/BOINC/boinc/blob/66c367e11e05712bdc04b7eeabaf522e5f64fe9f/clientgui/MainDocument.cpp#L2553 for reference

Reply:
This requested change needs to be addresses in order to move this PR forward.